test/e2e/kubernetes/kubernetes_test.go

```diff
@@ -2146,7 +2148,7 @@ var _ = Describe("Azure Container Cluster using the Kubernetes Orchestrator", fu
 						break
 					}
 				}
-
+				// TODO refactor to remove the "compute" usage so the test can be run on Azure Stack
```

Comment:
We're O.K. with leaving this TODO as is? This means these tests will fail, right?

Reply:
(fail in Azure Stack, that is)

Reply:
OK, for now, it wont get to this point as we do not yet support VMSS

Reply:
https://github.com/Azure/aks-engine/pull/3371/files/7910d7b76b9f89603208235636059dc2deb76d68#diff-3509723ee1508f842e543bf659b43e20R2067